Vault-Deployment/values/kube-prometheus-stack-values.yaml

```diff
@@ -881,6 +880,28 @@ grafana:
   #   url: https://{{ printf "%s-prometheus.svc" .Release.Name }}:9090
   #   version: 1
 
+  # https://grafana.com/docs/grafana/latest/datasources/loki
+  - name: Promtail
```

Comment:
@eyenx we configure two Grafana datasources here, one for "Kubernetes logs" and another one for the "Vault audit logs". As discussed, a pity that Grafana organizations are not yet configurable in Helm/yaml.. I decided not to go down that rabbit hole for the demo purposes here and simply add the datasources to the same org ID (`1`, the first/default org). But obviously, you could add them to different orgs and authorize accordingly. The `X-Scope-OrgID` is set to match what was set at the originating place (where the logs where forwarded initially, Promtail or OTLP). Otherwise, Grafana is unable to "read" the log stream.